README.md

```diff
@@ -12,7 +12,7 @@ Hello and welcome to this repository with examples showcasing the [TomTom Naviga
 
 Once you have gained access, set up the development environment by following:
 
-1. Install [Xcode](https://apps.apple.com/app/id497799835) 14 or higher if you don't already have it.
+1. Install Xcode if you don't already have it. Supported versions of Xcode are 13 to 14, inclusive.
```

Comment:
nit: I'd rather keep just 14.X, without extending the list of the possible issues for us. We don't test on Xcode 13, so I'm not sure how stable it's.
```suggestion
1. Install Xcode if you don't have it. Supported versions of Xcode are 13 to 14, inclusive.
```

Reply:
As part of the PR pipeline, we will validate both versions of Xcode.